.docker/ci/Dockerfile

```diff
@@ -1,23 +1,25 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
 MAINTAINER Dave Coleman dave@picknik.ai
```

Comment:
feel free to make yourself the maintainer :)

also, MAINTAINER is deprecated, please use:

    LABEL maintainer="mike@picknik.ai"

https://docs.docker.com/engine/reference/builder/#/maintainer-deprecated